tests/testthat/test_independence_corr_func.R

```diff
@@ -0,0 +1,27 @@
+test_that(
+  "Test Independence Corr Func in Stan agrees with R",
+  {
+    model <- compiled_site_inf_model
+    space_model_fxns <- spatial_fxns
+
+    corr_func_params <- list(
+      num_sites = 5
+    )
+
+    ind_corr_func_stan <-
+      space_model_fxns$functions$independence_corr_func(
+        corr_func_params$num_sites
+      )
+    ind_corr_func_r <- independence_corr_func(
+      corr_func_params
+    )
+    # both functions return correlation matrix using an
+    # independent correlation function
+
+    testthat::expect_equal(
```

Comment:
Again, may want `expect_almost_equal` here

Reply:
Ignore this, see https://github.com/CDCgov/ww-inference-model/pull/39#discussion_r1700816804